packages/react-scripts/template/README.md

```diff
@@ -205,6 +205,34 @@ class Button extends Component {
 export default Button; // Don’t forget to use export default!
 ```
 
+To import your own modules into other files, you can use relative paths by default. For example:
```

Comment:
I think this would work better as a separate section called “Absolute Imports”. I would put it right before “Can I Use Decorators?” so that it’s there, but doesn’t distract from the usual way of doing things.